layouts/sitemap.xml

```diff
@@ -2,14 +2,18 @@
 {{ $versionUsed := print "docs/" $.Site.Params.version}}
 {{ range .Data.Pages -}}
     {{ if not .Params.private -}}
-        {{ if or (in .Permalink $versionUsed) (not (in .Permalink "docs"))}}
+        {{ if or (in .RelPermalink $versionUsed) (not (in .RelPermalink "docs/"))}}
+            {{ $lastModDate := .Lastmod -}}
+            {{ if not .Lastmod -}}
+                {{ $lastModDate = now -}}
+            {{- end}}
```

Comment:
why not put that into an if/else instead of first setting and then checking?

Reply:
I have yet to try to determine if this assumption is valid. Still, I usually need to define variables outside the condition anyways, so in the end, I favor just checking the error case and handling it, as it is fewer lines, and from a logic perspective, I think it is easier to follow

Reply:
i also coud do somethink like following based on https://gohugo.io/functions/cond/ which is actually a pretty nice and sweet idea
```suggestion
            {{ cond .Lastmod .Lastmod  now }}
```

Reply:
> i also coud do somethink like following based on https://gohugo.io/functions/cond/ which is actually a pretty nice and sweet idea

or
```suggestion
            {{ $lastModDate := default .Lastmod now }}
```

checking my options

Reply:
i would prefer the `default` oneliner :)